### DISCUSSION TITLE
getting "transaction_key": "deprecated" in the meta data response although key is new

### DISCUSSION THREAD

**Discussion by gilles437** [deepgram/community#260] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Hello,

For the last few days I have been using deepgram with a nodejs application.
I use a freshly generated API key and all worked fine until a few hours ago. But now all my requests fail.

I get now in my response transaction_key": "deprecated"  and I don't see a reason for that. I tried with other generated keys and it did not help.

The video I am uploading is ±3 mins long.

Is there some new requirements in the API? Any help would be much appreciated! 

My code is attached below.

Here is the full response:

{
  "metadata": {
    "transaction_key": "deprecated",
    "request_id": "81c83ef4-f148-4789-867b-f6693ebc5817",
    "sha256": "031541132b13a6163711790f7be8decb773d22a2263c251ec79b7ee6ab787a33",
    "created": "2023-07-05T15:57:50.111Z",
    "duration": 227.25299,
    "channels": 1,
    "models": [
      "3312b52b-d08e-4072-bfff-124c410b770d"
    ],
    "model_info": {
      "3312b52b-d08e-4072-bfff-124c410b770d": {
        "name": "general-nova",
        "version": "2023-03-13.31000",
        "arch": "nova"
      }
    }
  },
  "results": {
    "channels": [
      {
        "alternatives": [
          {
            "transcript": "Maitan. We",
            "confidence": 0.8111559,
            "words": [
              {
                "word": "maitan",
                "start": 22.154999,
                "end": 22.654999,
                "confidence": 0.44044513,
                "punctuated_word": "Maitan."
              },
              {
                "word": "we",
                "start": 25.677498,
                "end": 26.177498,
                "confidence": 0.8111559,
                "punctuated_word": "We"
              }
            ]
          }
        ]
      }
    ]
  }
}

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

81c83ef4-f148-4789-867b-f6693ebc5817

### If possible, please attach your code or paste it into the text box.

DEEPGRAM_KEY = '<PRIVATE KEY>'
const deepgram = new Deepgram(DEEPGRAM_KEY)

const file = {
        buffer: fs.readFileSync(videoFileName),
        mimetype: 'audio/mp3',
}
const options = {
        punctuate: true,}

const result = await deepgram.transcription
        .preRecorded(file, options)
        .catch((e) => console.log(e))

console.log('The result:', JSON.stringify(result))

### If possible, please attach an example audio file to reproduce the issue.

happens with any audio file

**Comment by jjmaldonis** [comment, marked as answer]
Hey @gilles437, you can ignore the `transaction_key: deprecated` field in the response. It is a relic of an old feature that we have deprecated. So whatever is going wrong is unrelated to that. You said your requests are now failing - that is likely to only occur if the audio has changed and/or the mimtype is incorrect. In this case, `audio/mp3` is not a valid mimetype; they use `audio/mpeg` for mp3s for some reason. Below is a table with common mimetypes:

mp3 - audio/mpeg
mp4 audio- audio/mp4
m4a - audio/mp4
wav - audio/vnd.wav
au - audio/basic
snd - audio/basic
Linear PCM - audio/L24
mid - audio/mid
rmi - audio/mid
aif - audio/x-aiff
aifc - audio/x-aiff
aiff - audio/x-aiff
m3u - audio/x-mpegurl
ra - audio/vnd.rn-realaudio
ram - audio/vnd.rn-realaudio
Ogg Vorbis - audio/ogg
Vorbis - audio/vorbis

**Reply by gilles437** [reply]
Thank you Jason for the prompt reply.
However, changing the mime type to audio/mpeg  doesn't help in this case

**Reply by gilles437** [reply]
And the audio didn't change, I tried also with several mp3 files that are correct

**Comment by gilles437** [comment]
Ok, I got the point.
Actually, the other videos that I tested were in english.
This time I ran the app with audio in french, and it appears that it works only with the model "whisper", but it doesn't work if I don't provide a model.

**Reply by jjmaldonis** [reply]
Got it, that makes sense. If you use `model=general` and `language=fr`, that will work as well. If a `language` isn't provided and `detect_language=true` is not enabled, then the language defaults to English.
